pkg/reconciler/v1alpha1/taskrun/resources/input_resource_test.go

```diff
@@ -1041,3 +1002,25 @@ func TestAddStepsToTaskWithBucketFromConfigMap(t *testing.T) {
 		})
 	}
 }
+
+func mockResolveTaskResources(taskRun *v1alpha1.TaskRun) map[string]v1alpha1.PipelineResourceInterface {
+	resolved := make(map[string]v1alpha1.PipelineResourceInterface)
+	for _, r := range taskRun.Spec.Inputs.Resources {
+		var i v1alpha1.PipelineResourceInterface
+		if name := r.ResourceRef.Name; name != "" {
+			i = inputResourceInterfaces[name]
+			resolved[r.Name] = i
+		} else if r.ResourceSpec != nil {
+			i, _ =v1alpha1.ResourceFromType(&v1alpha1.PipelineResource{
```

Comment:
i think some of your auto-formatting settings might be a bit off here cuz I would expect to see a ` ` after the `=` 🤔

Reply:
Ran gofmt!!